packages/node/src/dev-server.ts

```diff
@@ -35,14 +41,7 @@ function parseRuntime(
   const project = new Project();
   const staticConfig = getConfig(project, entryPointPath);
   const runtime = staticConfig?.runtime;
-  if (runtime && !isEdgeRuntime(runtime)) {
-    throw new Error(
-      `Invalid function runtime "${runtime}" for "${entrypoint}". Valid runtimes are: ${JSON.stringify(
-        Object.values(EdgeRuntimes)
-      )}. Learn more: https://vercel.link/creating-edge-functions`
-    );
-  }
```

Comment:
**question:** since this code was very close to [this one](https://github.com/vercel/vercel/pull/9286/files#diff-b80a0ec8c1900042621cb5f358ad6f6d54d114269c292d28b57146fe0ecf4b92L409), I merged them in an utility. Is it worthy?

In particular, the error message differ. Previously, in dev mode:
>  Invalid function runtime "bla" for "api/edge.js". Valid runtimes are: "edge", "experimental-edge". Learn more: https://vercel.link/creating-edge-functions`

Previously, at build time:
> Unsupported "runtime" property in `config`: "bla" (must be one of: "nodejs", "edge", "experimental-edge")

Now they both are:
> api/edge.js: unsupported "runtime" property in \`config\`: "bla" (must be one of: "nodejs", "edge", "experimental-edge"). Learn more: https://vercel.link/creating-edge-functions